makefiles/info-kconfig.inc.mk

```diff
@@ -0,0 +1,101 @@
+.PHONY: info-kconfig-board-supported-% info-kconfig-boards-supported
+
+ifneq (1,$(TEST_KCONFIG))
+
+info-kconfig-boards-supported:
+	@echo "To run $@ please set TEST_KCONFIG=1"
+
+info-kconfig-board-supported-%:
+	@echo "To run $@ please set TEST_KCONFIG=1"
+
+else
+
+# This is the root Kconfig
+KCONFIG ?= $(RIOTBASE)/Kconfig
+
+# Include tools targets
+include $(RIOTMAKE)/tools/kconfiglib.inc.mk
+
+# This file will contain the calculated dependencies formated in Kconfig
+export KCONFIG_GENERATED_DEPENDENCIES
+
+# This file will contain external module configurations
+export KCONFIG_EXTERNAL_MODULE_CONFIGS
+# This file will contain external package configurations
+export KCONFIG_EXTERNAL_PKG_CONFIGS
```

Comment:
I just included them for this not to fail, but we need to generate them to properly run the dependency check. What would be the best place to generate these files (keeping in mind that it is not for a specific board)?